### ISSUE TITLE
Peggle Deluxe Cant be both Full Screen and 3d Accelerated

### ISSUE THREAD

**Issue by TheRisingUnited** [the1812/Popcap-Patches#7]
When trying to go full screen the game blips but does nothing, If you turn 3d acceleration off it works though

**Comment by TheRisingUnited** [comment]
This also applies to Peggle Nights. Though now that game cannot be full screen at all even with 3d acceleration turned off

**Comment by the1812** [comment]
Yes, for Peggle series you can only choose one feature of them. I've updated the README file.
